config/service.go

```diff
@@ -144,6 +154,16 @@ func (c *ServiceConfig) Finalize() {
 
 	if c.Tag == nil {
 		c.Tag = String("")
+	} else {
+		log.Println("[WARN] (config) The 'tag' attribute was marked for " +
+			"deprecation in v0.2.0 and will be removed in a future version " +
```

Comment:
I can update this to a specific version if we know when we would want to remove the deprecated field.

Reply:
Great warning log for the deprecated tag field!